app/assets/stylesheets/components/_navigation.scss

```diff
@@ -1,10 +1,10 @@
-.rad__authentication {
+.rad__navigation {
```

Comment:
I know this is just a rename but what do you think about losing the `rad__` prefix? I'm unsure what purpose it serves as there doesn't appear to be a `rad` block that it's inheriting from? If it was due to a naming clash then `rad-navigation` would work better.